code/src/App.js

```diff
@@ -1,9 +1,34 @@
-import React from 'react'
+import React, { useState } from "react";
+
+import { Header } from "./components/Header";
+import { Footer } from "./components/Footer";
+import { ToggleQuestions } from "./components/ToggleQuestions";
 
 export const App = () => {
+  const [questionNumber, setQuestionNumber] = useState("");
+
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="wrapper" aria-sort="descending">
+      <header className="header" aria-labelledby="header">
+        <Header title="The FUTURE" />
+      </header>
+
+      {questionNumber === "" && (
+        <main>
+        <div className="start-page" aria-labelledby="start-heading">
+          <h2 className="question-heading" id="start-heading">Let's talk about the future!</h2>
+        </div>
+        </main>
```

Comment:
You could create something like `Welcome.js` to make your app.js only decides which components will be rendered.